ironfish/src/rpc/routes/wallet/multisig/createSigningPackage.test.ts

```diff
@@ -52,23 +45,29 @@ describe('Route multisig/createSigningPackage', () => {
       },
     }
 
-    const response = await routeTest.client.wallet.importAccount(importAccountRequest)
+    await routeTest.client.wallet.importAccount(importAccountRequest)
 
-    const commitments: Array<string> = []
-    for (let i = 0; i < 3; i++) {
-      const signingCommitment = await routeTest.client.wallet.multisig.createSigningCommitment({
-        account: response.content.name,
-        seed,
-      })
-      commitments.push(signingCommitment.content.commitment)
-    }
+    const txAccount = await useAccountAndAddFundsFixture(routeTest.wallet, routeTest.chain)
+    const unsignedTransaction = (
+      await useUnsignedTxFixture(routeTest.wallet, txAccount, txAccount)
+    )
+      .serialize()
+      .toString('hex')
+
+    const commitments = await Promise.all(
+      participants.map(async (_) => {
+        const signingCommitment =
+          await routeTest.client.wallet.multisig.createSigningCommitment({
+            unsignedTransaction,
+            signers: [], // TODO
```

Comment:
Does this need to be set?

Reply:
That needed to be set in some places (to make the tests more realistic). I forgot to resolve the TODOs before pushing; fixed